cutevariant/gui/selectionquerywidget.py

```diff
@@ -269,6 +269,15 @@ def save_current_query(self):
 
         if name != DEFAULT_SELECTION_NAME:
```

Comment:
Une liste reservé plutôt  if name not in (  "variants", "favoris" )

Reply:
à garder en tête pour cette fonctionnalité oui